### PR TITLE
Fix connection closed error for big files when loading interproscan

### PR DESCRIPTION
**Pull request by mboudet**
Should fix error such as 

```
psycopg2.OperationalError: server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
```

For big files (~20 Go)

Frankly, this is a band-aid fix, but it should work.
I did not modify the case where "pool_connections" is set to False (can we even set it from chakin?), since I'm not sure it's actually needed with `poolclass=NullPool`

### CODE REVIEW

**Comment by abretaud** [comment]
Thanks!
Not a problem when `pool_connections==False`: it's used for quick calls from the galaxy dynamic options.
Could you prepare a new release while you're at it? I guess we can also require python>=3.8 to fix the tests

**Comment by mboudet** [comment]
Sure, I'll look into it

File: setup.py [hunk 1]
```diff
@@ -31,4 +31,5 @@
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
```
Comment (abretaud) [inline setup.py]: Can you update this list? = start from 3.8 I guess

Comment (mboudet) [reply setup.py]: Sure.

**Pull request merged**